mlir/lib/Dialect/Tosa/IR/TosaCanonicalizations.cpp

```diff
@@ -73,56 +73,187 @@ struct SqrtReciprocalOptimization : public OpRewritePattern<tosa::PowOp> {
                                 PatternRewriter &rewriter) const override {
     // Check that the PowOp has a single user
     if (!op->hasOneUse())
-      return rewriter.notifyMatchFailure(op, "pow operator has more than one user");
+      return rewriter.notifyMatchFailure(op,
```

Comment:
nit: I see a lot of formatting issues that are unrelated to this change. I like to use `git add -p` to add chunks of my diff at a time selecting which ones I want to be included in the commits. Just to make sure I have the changes I want.